doc/userguide/file-extraction/file-extraction.rst

```diff
@@ -25,7 +25,7 @@ Settings
 
 *stream.checksum_validation* controls whether or not the stream engine rejects packets with invalid checksums. A good idea normally, but the network interface performs checksum offloading a lot of packets may seem to be broken. This setting is enabled by default, and can be disabled by setting to "no". Note that the checksum handling can be controlled per interface, see "checksum_checks" in example configuration.
 
-*file-store.stream-depth* controls how far into a stream reassembly is done. Beyond this value no reassembly will be done. This means that after this value the HTTP session will no longer be tracked. By default a settings of 1 Megabyte is used. 0 sets it to unlimited. If set to no, it is disabled and stream.reassembly.depth is considered.
+*file-store.stream-depth* controls how far into a stream reassembly is done. Beyond this value no reassembly will be done. This means that after this value the HTTP session will no longer be tracked. By default a setting of 1 Megabyte is used. 0 sets it to unlimited. If set to no, it is disabled and stream.reassembly.depth is considered. Non-zero values must be greater than ``stream.stream-depth`` to be used.
```

Comment:
Is this correct? If file-store.stream-depth is commented out it will use a default of 1 megabyte? I do find that a bit misleading. If its uncommented I would expect the global stream depth to be used and not some file-store specific default.  I think it could lead to confusion.

Reply:
Perhaps my wording is unnecessarily confusing? Here's the rules
* 0 -- no limit
* non-zero -- must be greater than `stream.depth` to be used; else, `stream.depth` is used.

My wording is incorrect by referring to `stream.reassembly.depth` and `stream.stream-depth` -- these refer to `stream.depth`